jetbrains-rider/protocol/model/daemon/LambdaDaemonModel.kt

```diff
@@ -46,6 +46,9 @@ object LambdaDaemonModel : Ext(SolutionModel.Solution) {
     }
 
     init {
+        setting(Kotlin11Generator.Namespace, "software.aws.toolkits.jetbrains.protocol")
+        setting(CSharp50Generator.Namespace, "AWS.Toolkit.Rider.Model")
```

Comment:
Do we need comments on what these mean?

Reply:
ya im still tweaking them. It broke 201,202

Reply:
Okay fixed, they are the kotlin package and C# namespace to put our code into